### PR TITLE
Add suport operators && and ||

### PR DESCRIPTION
**Pull request by janczer**
Relate to #55

### CODE REVIEW

File: vm/boolean_test.go [hunk 1]
```diff
@@ -41,6 +41,14 @@ func TestEvalInfixBooleanExpression(t *testing.T) {
...
 		{"(1 > 2) == true", false},
 		{"(1 > 2) == false", true},
+		{"true && true", true},
+		{"false && true", false},
+		{"true && false", false},
+		{"false && false", false},
+		{"true || true", true},
+		{"false || true", true},
+		{"true || false", true},
+		{"false || false", false},
```
Comment (st0012) [inline vm/boolean_test.go]: @janczer Can you add more complex condition to test new precedence setting? Like:

```ruby
100 > 10 && true == false # false
```

**Comment by st0012** [comment]
LGTM, just need more tests for new precedence 😄

**Comment by janczer** [comment]
How I can test new precedence?

File: vm/boolean_test.go (hunk 1 above)
Comment (st0012) [inline vm/boolean_test.go]: @janczer Here's the example for testing precedence.

Comment (janczer) [inline vm/boolean_test.go]: @st0012 Yes, I can. Thank you

**Pull request merged**